src/routes/resources/joining/+page.svelte

```diff
@@ -12,31 +13,44 @@
 		resources to opportunities for growth. Your participation not only enriches your experience but
 		also helps our chapter give resources and expeience that you can use. We are diligently working
 		towards inproving all student's education and experience, and your involvement is instrumental
-		in this endeavor.
-	</p>
-	<p>
-		If you are interested in joining ACM you can simply attend one our meetings or talk to anyone
-		that you know is associated with ACM about joining. If our meeting times don't work with you,
-		you can still participate in ACM with our subcommittees.
-	</p>
-	<p>
-		<b>Membership dues are $20 for one school year.</b> They're are not required but it does help out
-		the club to afford T-shirts and equipment.
-	</p>
-	<p>
-		Please make a myACM account, it's free and it'll continue supporting our chapter. <b>
-			Use this link to sign up:</b
-		>
-		<a
-			href="https://accounts.acm.org/"
-			class="text-blue-900 font-bold underline hover:text-blue-700"
-			>myACM
-		</a>
-	</p>
-	
-	<p>
-		Once you have successfully made an account, please forward your ACM ID number to either the
-		President or Secretary. Your contribution is pivotal in helping us achieve our membership goals
-		and further our impact as a chapter.
+		in this endeavor. <span>Membership dues are $20 for one school year.</span> They are not required
+		but it does help the club afford T-shirts and equipment.
 	</p>
+
+	<div class="py-4 grid gap-x-4 gap-y-20 md:grid-cols-2 sm:gap-y-16">
```

Comment:
you can replace "grid gap-x-4 gap-y-20 md:grid-cols-2 sm:gap-y-16" with "md:flex md:justify-between". The less we deal with absolute numbers the better unless we are dividing things when it comes to placing things.

Reply:
okay